Form/Type/MediaType.php

```diff
@@ -77,14 +76,34 @@ public function buildView(FormView $view, FormInterface $form, array $options)
 
     /**
      * {@inheritdoc}
+     *
+     * @deprecated To be removed when bumping to Symfony 2.7+, use MediaType::configureOptions() instead
      */
     public function setDefaultOptions(OptionsResolverInterface $resolver)
     {
+        if ($resolver instanceof OptionsResolver) {
```

Comment:
It prevents a fatal error while let both methods coexist (even with different behaviors if it needed) before bumping Symfony's version.
By insntance, a possible exception could be:
```php
Argument 1 passed to Sonata\MediaBundle\Form\Type\MediaType::configureOptions() must be an instance of Symfony\Component\OptionsResolver\OptionsResolver, MyOwnOptionsResolver given
```